Minor tweaks to be more consistent across objects

millen1m opened the pull request:
The main change is switching the optional indicator to always be within <> instead of [].
Minors corrections to some inputs to allow them to run.

zhuminjie commented:
@millen1m Thanks!

Everything looks good. Just to clarify few things:

- the new file src/ExplicitDifference.rst seems a copy of existing file src/explicitDifference.rst
- src/fiberThermalsection.rst, the reason to use :doc: instead of toctree is that both fiberSection and fiberThermalSection contains the same subsection, which seems not able to be included in two subtree.

millen1m commented:
I have now resolved the duplicate ExplicitDifference file issue and reverted the fibreThermalSection file.
I have also changed the mention of slave and master nodes to constrained and retained nodes - consistent with the changes to the OpenSees.berkeley.edu documentation.

zhuminjie commented:
Thank you!
